server/src/test/java/e2e/UserTaskTest.java

```diff
@@ -102,6 +124,20 @@ public Workflow buildDeadlineReassignmentWorkflowUserWithoutGroup() {
         });
     }
 
+    @LHWorkflow("deadline-reassignment-with-epoch")
+    public Workflow buildDeadlineReassignmentWithEpoch() {
+        return new WorkflowImpl("deadline-reassignment-with-epoch", entrypointThread -> {
+            entrypointThread.spawnThread(
+                    utThread -> {
+                        UserTaskOutput formOutput = utThread.assignUserTask(USER_TASK_DEF_NAME, "yoda", "my-group");
+                        utThread.releaseToGroupOnDeadline(formOutput, 20);
+                    },
+                    "user-task-thread",
+                    Map.of());
+            entrypointThread.spawnThread(sleepThread -> sleepThread.sleepSeconds(6), "sleep-thread", Map.of());
```

Comment:
Can we use `1` second here, and only `3` seconds on line 133? Otherwise it will make the pipeline much slower.

Reply:
see
https://github.com/littlehorse-enterprises/littlehorse/pull/667#discussion_r1492914451